react/Avatar/index.jsx

```diff
@@ -60,23 +61,34 @@ export const Avatar = ({
       {!image && text && (
         <span className={styles['c-avatar-initials']}>{text}</span>
       )}
-      {!image && !text && <Icon icon="people" />}
+      {!image && !text && <Icon icon={icon} />}
     </div>
   )
 }
 
 Avatar.propTypes = {
   text: PropTypes.string,
   image: PropTypes.string,
-  size: PropTypes.oneOf(['xsmall', 'small', 'medium', 'large', 'xlarge']),
-  className: PropTypes.string
+  size: PropTypes.oneOf([
+    'xsmall',
+    'smaller',
+    'small',
+    'medium',
+    'large',
+    'xlarge'
+  ]),
```

Comment:
Hard to follow all this size. Why not starting to add "size=24" instead of "smaller" ?

Reply:
Yes. @joel-costa  do we really need 6 sizes of Avatar ? I think we should have less sizes and the ability for the user to give a custom size.

Reply:
@ptbrowne : Size doesn't matter for you ? :p
You would have only 3 sizes like on MUI ? No worries for me if we can give a custom size. Even on the home, we have a specific size : 
![image](https://user-images.githubusercontent.com/36403934/98550240-25d6aa80-229c-11eb-87ef-f6c543bf0e0b.png)

But I don't know if it would be relevant to set the same 3 sizes as MUI (24 - 40 - 56). 
If we exclude the home, we only use 16 - 24 - 32 - 40 right ?